SOLID-violation-examples.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
I think this is your local file. It doesn't have to be pushed to the remote repo.